lib/model/frames/entity.js

```diff
@@ -20,7 +20,8 @@ class Entity extends Frame.define(
   'createdAt', readable,    'creatorId', readable,
   'updatedAt', readable,    'deletedAt', readable,
   embedded('creator'),
-  embedded('currentVersion')
+  embedded('currentVersion'),
+  embedded('dataset') // used on occasion when entire dataset attached to entity
```

Comment:
Can remove: I added this when I was trying to reconstruct a full Entity w/ source and everything from the insert query. I also needed to add the dataset to it, so I could use the dataset actee id in the audit logging, and also have the `.forApi()` function happily format the entire Entity as the right API response. 

This part of the code was getting complex, there were a bunch of different ways to do things, none which felt great. I ended up just re-fetching the entity from the database so all the info would be there in the right way.